src/setupTests.ts

```diff
@@ -3,3 +3,5 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import "@testing-library/jest-dom";
+
+global.ResizeObserver = require("resize-observer-polyfill");
```

Comment:
So why exactly are you replacing the `ResizeObserver` with a polyfill? Seems to me [it's supported by all relevant browsers](https://caniuse.com/resizeobserver).

Reply:
It's only for testing purposes

Reply:
ok makes sense 👍

Reply:
(for anyone else wondering, this is the reason: https://github.com/ZeeCoder/use-resize-observer/issues/40)